Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -263,7 +295,8 @@ final class CustomerCenterViewModelTests: TestCase {
         expect(purchaseInformation.title) == "title"
         expect(purchaseInformation.durationTitle) == "1 month"
 
-        expect(purchaseInformation.pricePaid) == .nonFree("$2.99")
+        expect(purchaseInformation.pricePaid) == .nonFree(formatted(price: 3.99, currencyCode: currency))
+        expect(purchaseInformation.renewalPrice).to(beNil())
```

Comment:
We can also mock a RenewalInfo here to have a `renewalPrice`?

Reply:
We can, I've done the two combinations in one test to prove it works. 

After fixing all of them, I realised there's a dedicated test for it already `testPurchaseInformationUsesInfoFromRenewalInfoWhenAvailable`. 😅